File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/notification/UnknownNotification.java
package com.bitlove.fetlife.notification;

import android.content.Context;

import com.bitlove.fetlife.FetLifeApplication;

import org.json.JSONObject;

@Deprecated
public class UnknownNotification extends OneSignalNotification {

    public UnknownNotification(String title, String message, String launchUrl, JSONObject additionalData, String id, String group) {
        super(id, title, message,launchUrl, group, additionalData);
    }

    @Override
    public boolean handle(FetLifeApplication fetLifeApplication) {
        //Hide notification
        return true;
    }

    @Override
    public void onClick(FetLifeApplication fetLifeApplication) {
    }

    @Override
    public String getAssociatedPreferenceKey(Context context) {
        return null;
    }
}

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/inbound/onesignal/notification/UnknownNotification.kt
package com.bitlove.fetlife.inbound.onesignal.notification

import com.bitlove.fetlife.FetLifeApplication
import com.bitlove.fetlife.R
import com.bitlove.fetlife.notification.NotificationParser
import org.json.JSONObject

class UnknownNotification(id: String, title: String, message: String, launchUrl: String, group: String, additionalData: JSONObject) : OneSignalNotification("unknown", id, title, message, launchUrl, group, additionalData, -1) {

    //Hide notification
    override fun handle(fetLifeApplication: FetLifeApplication): Boolean {
        return true
    }

}